.gitlab/image.yml

```diff
@@ -72,15 +72,13 @@
   image: ginkgohub/rocm:45-mvapich2-gnu8-llvm8
   tags:
     - private_ci
-    - amdci
-    - gpu
+    - amd-gpu
```

Comment:
Are these changes from the PR #1394 ?

Reply:
no, I just noticed we are barely using nla1, and this enables more jobs to be run there.